docs/listings/hello2.txt

```diff
@@ -0,0 +1,50 @@
+[ReFrame Setup]
+  version:           3.10.0-dev.2+bf404ae1
+  command:           './bin/reframe -c tutorials/basics/hello/hello2.py -r'
+  launched by:       karakasv@vpn-39.cscs.ch
+  working directory: '/Users/karakasv/Repositories/reframe'
+  settings file:     '<builtin>'
+  check search path: '/Users/karakasv/Repositories/reframe/tutorials/basics/hello/hello2.py'
+  stage directory:   '/Users/karakasv/Repositories/reframe/stage'
+  output directory:  '/Users/karakasv/Repositories/reframe/output'
```

Comment:
I assume you want to change karakasv to user like in the other listings.

Reply:
Yes, I've eventually addressed #2391 in this PR and now all listings are generated correctly automatically.